src/validations/rules/poam.sch

```diff
@@ -210,6 +210,7 @@
             <sch:assert
                 diagnostics="has-ssp-rlink-diagnostic"
                 doc:guide-reference="Guide to OSCAL-based FedRAMP Plan of Action and Milestones (POA&amp;M) §3.5"
+                fedramp:specific="true"
```

Comment:
I'm not going to go any farther than this one.

How can a POA&M reference to an SSP be FedRAMP specific?